Simplified/NYPLBookDetailViewController.m

```diff
@@ -193,8 +193,15 @@ - (void)problemReportViewController:(NYPLProblemReportViewController *)problemRe
 {
   NSURL *reportURL = problemReportViewController.book.reportURL;
   if (reportURL) {
-    NSURLRequest *r = [NSURLRequest postRequestWithProblemDocument:@{@"type":type} url:reportURL];
-    [[NYPLSession sharedSession] uploadWithRequest:r completionHandler:nil];
+    NSURLRequest *request = [NSURLRequest
+                             postRequestWithProblemDocument:@{@"type":type}
+                             url:reportURL];
+    NYPLNetworkExecutor *executor = [[NYPLNetworkExecutor alloc]
```

Comment:
I'm unsure if this can work reliably, because the `executor` instance goes out of scope right after the POST call. Unless there is something that binds the request to a global queue or other object, but I don't think our Executor stack does that.

A strongly retained property sounds more reliable to me.

Reply:
Done